src/main/resources/category/java/common.xml

```diff
@@ -1416,22 +1416,31 @@ public class Good {
                 <value><![CDATA[
 (: append used in one chained statement on constructor of StringBuilder and with chained toString :)
 //MethodDeclaration//Block//MethodCall[@MethodName = "toString"]/MethodCall[@MethodName='append']
-[.//ConstructorCall[pmd-java:typeIs('java.lang.StringBuilder') or pmd-java:typeIs('java.lang.StringBuffer')]]
+[.//ConstructorCall[pmd-java:typeIs('java.lang.AbstractStringBuilder')]]
```

Comment:
interesting (unexpected) and good to know that `StringBuffer` also extends `AbstractStringBuilder`